### DISCUSSION TITLE
Exporting large albumn via CLI

### DISCUSSION THREAD

**Discussion by chengguangnan** [immich-app/immich#11496] [General]
I have some large vaction album, which has about 8k photos and videos sizing around 100GB, which I want to export and move to my Andorid phone so I can browse fast and offline. 

I'm using the web ui to download it. So far chrome made 10+ zip files of about 5GB each.

My question is:

Is there a better way to export a large albumn? 

Maybe use the CLI to download an album?

**Comment by chengguangnan** [comment]
I wrote a script to download album.

https://github.com/chengguangnan/immich-ruby/tree/main

**Comment by NiklasRosenstein** [comment]
Building on @chengguangnan's script, I made a version that is more to my liking. It produces a single output directory for an album and keeps track of already downloaded assets so you can easily resume the download.

https://gist.github.com/NiklasRosenstein/91d1bfc22485a4530271959300a69acf
